batavia/types/List.js

```diff
@@ -78,8 +77,17 @@ batavia.types.List = function() {
 
         if (other !== null) {
             if (batavia.isinstance(other, batavia.types.List)) {
-                /* update this line to get Pythonic list < list behavior */
-                return this.valueOf() < other;
+                // edge case where this=[]
+                if(this.length === 0 && other.length > 0){return true}
```

Comment:
Stylistically, this sort of compression isn't desirable.
```
if (this.length === 0 && other.length > 0) {
    return true;
}
```
* space between `if` and the opening brace
* K&R style brackets, one statement per line semicolon terminated.

We don't need to save bytes or lines of code, and compressing like this impedes readability.